tests/client/test_workspaces.py

```diff
@@ -17,16 +17,121 @@
 from argilla.client import api
 from argilla.client.workspaces import Workspace
 
+from tests.helpers import SecuredClient
+
+
+def test_workspace_from_name(mocked_client: SecuredClient) -> None:
+    the_api = api.active_api()
+    workspace = the_api.http_client.post("/api/workspaces", json={"name": "test_workspace"})
+    assert workspace["name"] == "test_workspace"
```

Comment:
You can setup your test by using the DB factories.
```suggestion
    workspace = WorkspaceFactory.create(name="test_workspace") # this will create a workspace in DB that is accessible through the API
```

Reply:
Right, I'll refactor the unit tests then!